src/components/UserProfile/SocialLinks.jsx

```diff
@@ -83,7 +82,7 @@ class SocialLinks extends React.Component {
                 content="Social Links"
                 showEditButton
                 onClickEdit={() => onEdit('socialLinks')}
-                showVisibility={Boolean(socialLinks.length)}
+                showVisibility={Boolean(socialLinks && socialLinks.length)}
```

Comment:
Stylistically I'd probably just do `socialLinks && socialLinks.length > 0` here and avoid the `Boolean`...

Reply:
Good point that was leftover from elsewhere.